en/docs/chapter_sorting/bubble_sort.md

```diff
@@ -1,8 +1,8 @@
 # Bubble sort
 
-<u>Bubble sort</u> achieves sorting by continuously comparing and swapping adjacent elements. This process resembles bubbles rising from the bottom to the top, hence the name bubble sort.
+<u>Bubble sort</u> works by continuously comparing and swapping adjacent elements. This process is reminiscent of bubbles rising from the bottom to the top, hence the name “bubble sort.”
```

Comment:
I think "reminiscent of" used here is too official. How about "This process is like bubbles rising from the bottom to the top, hence the name bubble sort."